fiware-region-sanity-tests/README.rst

```diff
@@ -58,94 +58,103 @@ Test Algorithm
+* Test whether it is possible to deploy an instance with new keypair
+* Test whether it is possible to deploy an instance with new security group
+* Test whether it is possible to deploy an instance with all params
+* Test whether it is possible to deploy and instance and assign an allocated
+  public IP
+* Test whether it is possible to deploy and instance, assign an allocated
+  public IP and establish a SSH connection
 
 
-Configuration file
-------------------
+Configuration
+-------------
 
-Some configuration is needed before test case execution. This configuration is
-set in the ``resources/settings.json`` file:
+Some configuration is needed before test execution. This configuration may come
+from the file ``resources/settings.json`` or from the following environment
+variables (which override values from such file):
 
-- FIWARE Keystone endpoint.
-- Valid FIWARE credentials for the configured *keystone_url*: User, Password and TenantId.
-- Some configuration about each region: External Network name
+* ``OS_AUTH_URL``
+* ``OS_USERNAME``
+* ``OS_PASSWORD``
+* ``OS_TENANT_ID``
+* ``OS_TENANT_NAME``
 
-All configuration values will be 'strings'.
+Apart from the former data needed for authorization (``credentials`` section in
+settings file), it is possible to provide some per-region configuration values
+under ``region_configuration``:
 
-**Environment configuration example** ::
+* ``external_network_name`` is the network for external floating IP addresses
+* ``test_flavor`` let us customize the flavor of instances launched in tests
+
+
+**Configuration example** ::
 
     {
         "environment": "fiware-lab",
         "credentials": {
             "keystone_url": "http://cloud.lab.fiware.org:4731/v2.0/",
             "tenant_id": "00000000000000000000000000000",
-            "tenant_name": "myTenantName",
+            "tenant_name": "MyTenantName",
             "user": "MyUser",
             "password": "MyPassword"
         },
         "region_configuration": {
             "external_network_name": {
-                "Spain": "public-ext-net-01",
-                "Spain2": "public-ext-net-01",
-                "Trento": "public-ext-net-01",
-                "Lannion": "public-ext-net-02",
-                "Waterford": "public-ext-net-01",
-                "Berlin": "public-ext-net-01",
-                "Prague": "default",
-                "Mexico": "ext-net",
-                "PiraeusN": "public-ext-net-01",
-                "PiraeusU": "public-ext-net-01",
-                "Zurich": "public-ext-net-01",
-                "Karlskrona": "public-ext-net-01",
-                "Volos": "public-ext-net-01",
-                "Budapest": "public-ext-net-01",
-                "Stockholm": "public-ext-net-01",
-                "SophiaAntipolis": "net04_ext",
-                "Poznan": "public-ext-net-01",
-                "Gent": "public-ext-net-03",
-                "Crete": "public-ext-net-01"
```

Comment:
If you delete this value make a reference to the document in which a user can find the values due to they are using it like reference to understand what they want to do.